lib/reporting/identity_verification_report.rb

```diff
@@ -337,23 +382,40 @@ def query
         ),
         idv_final_resolution: quote(Events::IDV_FINAL_RESOLUTION),
         fraud_review_passed: quote(Events::FRAUD_REVIEW_PASSED),
+        fraud_event_names: quote(
+          [
+            Events::FRAUD_REVIEW_PASSED,
+            Events::FRAUD_REVIEW_REJECT_AUTOMATIC,
+            Events::FRAUD_REVIEW_REJECT_MANUAL,
+          ],
+        ),
+        normalized_fraud_review_pending: "coalesce(#{[
+          # rubocop:disable Layout/LineLength
+          'properties.event_properties.fraud_review_pending',
+          'not isblank(properties.event_properties.fraud_pending_reason)',
+          'properties.event_properties.fraud_check_failed',
+          '(ispresent(properties.event_properties.tmx_status) and properties.event_properties.tmx_status not in ["threatmetrix_review", "threatmetrix_reject"])',
+          '0',
+          # rubocop:enable Layout/LineLength
+        ].join(", ")})",
       }
 
       format(<<~QUERY, params)
         fields
             name
           , properties.user_id AS user_id
           , coalesce(properties.event_properties.success, 0) AS success
-        #{issuers.present? ? '| filter properties.service_provider IN %{issuers}' : ''}
+          , coalesce(properties.service_provider, properties.event_properties.issuer) AS service_provider
         | filter name in %{event_names}
-        | filter (name = %{usps_enrollment_status_updated} and properties.event_properties.passed = 1 and properties.event_properties.tmx_status not in ['threatmetrix_review', 'threatmetrix_reject'])
+        | filter (name = %{usps_enrollment_status_updated} and properties.event_properties.passed = 1)
                  or (name != %{usps_enrollment_status_updated})
-        | filter (name in %{gpo_verification_submitted} and properties.event_properties.success = 1 and !properties.event_properties.pending_in_person_enrollment and !properties.event_properties.fraud_check_failed)
+        | filter (name in %{gpo_verification_submitted} and properties.event_properties.success = 1 and !properties.event_properties.pending_in_person_enrollment)
                  or (name not in %{gpo_verification_submitted})
         | filter (name = %{fraud_review_passed} and properties.event_properties.success = 1)
                  or (name != %{fraud_review_passed})
+        #{issuers.present? ? '| filter service_provider IN %{issuers} OR name IN %{fraud_event_names}' : ''}
         | fields
-            coalesce(properties.event_properties.fraud_review_pending, properties.event_properties.fraud_pending_reason, 0) AS fraud_review_pending
+            %{normalized_fraud_review_pending} AS fraud_review_pending
           , coalesce(properties.event_properties.gpo_verification_pending, 0) AS gpo_verification_pending
           , coalesce(properties.event_properties.in_person_verification_pending, 0) AS in_person_verification_pending
           , ispresent(properties.event_properties.deactivation_reason) AS has_other_deactivation_reason
```

Comment:
Summarizing changes to the Cloudwatch query:
1. Normalize `service_provider` across a couple of field names
2. Don't filter out USPS events marked for fraud
3. Don't filter out GPO verification events marked for fraud
4. When filtering by issuer, don't filter out fraud events
5. Normalize fraud_review_pending, which goes by like 500,000 different names